exercises/practice/sgf-parsing/tests/Tests.elm

```diff
@@ -24,11 +24,17 @@ tests =
                 \() ->
                     SgfParsing.parse ";"
                         |> Expect.equal (Err "tree missing")
+
+        -- this test is structured a bit differently to avoid an elm-review NoUnused.CustomTypeConstructorArgs error
```

Comment:
I'm not suuuuper happy with this change, but it's better than ignoring the rule altogether for this exercise. For info, here is the full error:

```
-- ELM-REVIEW ERROR ------------------------------------ src/SgfParsing.elm:8:12

NoUnused.CustomTypeConstructorArgs: Argument is never extracted and therefore
never used.

7| type Tree
8|     = Node { properties : Dict String (List String), children : List Tree }
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

This argument is never used. You should either use it somewhere, or remove it at
the location I pointed at.
```

In a real project, that would highlight that no one is actually using this data structure, which is fair.

Reply:
I think this is fine.